packages/analytics/__tests__/analytics.test.ts

```diff
@@ -98,6 +98,31 @@ describe('Analytics', function () {
     );
   });
 
+  it('throws if consentSettings is not an object', function () {
```

Comment:
If I had to prefer one spot - the testing should be on the modular APIs (which is likely a nearly verbatim hunk of tests in the second half of this file ? - but testing in both is great. What's missing either way is the modular API test though